scripts/initialize_database.py

```diff
@@ -127,9 +124,9 @@ def run():
 
     if file_exists(db_file_path):
         print(f"Database found at {db_file_path}..")
-        ret_setup = setup_database(db_file_path)
-        if not ret_setup:
-            print(f"Error! Database setup at {db_file_path}..")
+        ret_load = load_database(db_file_path)
```

Comment:
As mentioned earlier, let's use `try`/`except` rather than return codes. I see the true/false style used elsewhere in this module -- let's use try/except everywhere